mapping/requirements_officejs.json

```diff
@@ -1948,11 +1948,34 @@
 						{
 							"name": "DialogApi",
 							"apiVersion": "1.1",
+							"supportedProductVersions": [{
```

Comment:
I suppose this section is for WAC, and no version should be required, right?

Reply:
WAC doesn't require any version#

Reply:
Sounds like this PR is not needed then.